authors/tests/base_test.py

```diff
@@ -19,11 +19,13 @@ def setUp(self):
         """This method is used to initialize variables used by the tests."""
 
         self.client = APIClient()
+        self.unauthenticated_client = APIClient()
```

Comment:
This object is equal to the `self.client` instance of the `APIClient` class. Is there a particular reason why you sought to add it, if not, please remove it and use the initial one.

Reply:
I added this because I realized that in the tests, once I call the method that creates the article, the verified user still remains logged in, and there was no way to log him out. For testing endpoints such as the one I've captured below, I had to use another client:

<img width="580" alt="screenshot 2019-01-30 at 19 26 15" src="https://user-images.githubusercontent.com/23090268/51996003-fb4f9f00-24c4-11e9-99b7-4597033620a6.png">